test/E2ETest/Microsoft.Test.E2E.AspNet.OData/ComplexTypeInheritance/ComplexTypeInheritanceDataModels.cs

```diff
@@ -49,7 +49,7 @@ public class Rectangle : Polygon
         public Point TopLeft { get; set; }
         public int Width { get; set; }
         public int Height { get; set; }
-        
```

Comment:
remove the whitespace

Reply:
It seems nothing changed in this file, please "undo" it.

---
In reply to: [191285099](https://github.com/OData/webapi/pull/1445#discussion_r191285099) [](ancestors = 191285099)

Reply:
Unintentional whitespace updates in related file only. I think what happen is that I made some edits and undid it later among local commits. Trailing whitespace were cleaned up automatically as part of this process. Let's include related trailing whitespace cleanup like this since we always cleanup trailing whitespace when a file is updated?

Here is a whitespace auto-cleanup tool you can use with VS in case you are interested: [https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer), feel free to let me know if you have any questions.

---
In reply to: [191285124](https://github.com/OData/WebApi/pull/1445#discussion_r191285124) [](ancestors = 191285124,191285099)